src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.interfaces.AreaCalculator;
+
+public class Circle extends Figure implements AreaCalculator {
+    private static final String TYPE_FIGURE = "CIRCLE";
```

Comment:
you don't need such constant, remove it, the same for other figures, you can use directly in method enum value instead